src/utilities/_hypre_utilities.hpp

```diff
@@ -1481,14 +1487,21 @@ void __syncwarp()
 {
 }
 
-#endif // #if defined(HYPRE_USING_HIP) || (CUDA_VERSION < 9000)
+static __device__ __forceinline__
+hypre_int __any_sync(unsigned mask, hypre_int predicate)
+{
+   return __any(predicate);
```

Comment:
Do you need `HYPRE_UNUSED_VAR(mask);` here?

Reply:
Yes, thanks for catching. I added it to a few other places as well